.changeset/curly-oranges-end.md

```diff
@@ -0,0 +1,6 @@
+---
+"@astrojs/cloudflare": minor
+"@astrojs/netlify": minor
+---
+
+Updates internal use of `app.render()` to use new signature
```

Comment:
If I'm not mistaken, this is the upstream PR with similar changes? https://github.com/withastro/astro/pull/9199/

I'd take a look at those changesets and see if any of those comments are relevant here. Notice how the changeset for the other adapters specifically mentions that you need to upgrade to Astro 4.0. Would that be the same here?

I might also include, since this major change is not happening in the same PR this time, linking back to that PR saying that it is upgrading for the Astro 4.0 changes made there.

Reply:
@sarah11918 Thanks for the comment, makes total sense to me. And you are right!

I wrote the changeset, so I'll update it!